broker/broker-route-manager/pom.xml

```diff
@@ -103,7 +103,7 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.68</minimum>
```

Comment:
❓Do you think we should keep those as decimals for now everywhere until we decide to switch to `%` everywhere to be consistent?

Reply:
I'm definitely open to that, I mainly switched these so people at least knew it was a new option.

Reply:
I changed it back for consistency purposes